qa/L0_backend_identity/identity_test.py

```diff
@@ -173,8 +171,9 @@
                 input_data = (16384 * np.random.randn(*shape)).astype(np_dtype)
             else:
                 in0 = (16384 * np.ones(shape, dtype='int'))
-                in0n = np.array([str(x) for x in in0.reshape(in0.size)],
-                                dtype=object)
+                in0n = np.array(
+                    [str(x).encode('utf-8') for x in in0.reshape(in0.size)],
```

Comment:
Why is explicit utf-8 encoding needed? Is it because python add-sub is expecting that?

Reply:
This is not Python add_sub. This is the L0_backend_identity test. It was required before too but it was performed in the client code. Previously, whenever the type of the ndarray was `np.object` it was encoded using UTF-8. The client does not require string to be encoded in UTF-8 but this is what the model expects. Also, because the HTTP client needs the final string to be JSON serializable, it requires it to be UTF-8 encoded (or encodable). But in the client code itself, we don't require the byte array to be encoded in UTF-8.